### DISCUSSION TITLE
GraphRAG for IA

### DISCUSSION THREAD

**Discussion by SlavaKeshkov** [microsoft/PubSec-Info-Assistant#872] [Ideas]
Microsoft recently released new RAG approach on Graph Data. Is there any idea to use this approach in PubSec assistant for more holistic document understanding? 

https://www.microsoft.com/en-us/research/blog/graphrag-unlocking-llm-discovery-on-narrative-private-data/
https://github.com/Azure-Samples/graphrag-accelerator

Thanks!

**Comment by dayland** [comment]
Not at the moment, but we are keeping a eye on the Graph RAG repo and processes.
